source/userguide/getting_started.rst

```diff
@@ -487,6 +487,11 @@ Troubleshooting
 |               |                                             | increase memory to 4GB from                   |
 |               |                                             | *Preferences | Advanced*.                     |
 +---------------+---------------------------------------------+-----------------------------------------------+
+| Elasticsearch | ``KnowledgeBaseConnectionError``            | If the error is due to maximum                |
```

Comment:
1. The error type should be `KnowledgeBaseError` rather than `KnowledgeBaseConnectionError`?
2. On the resolution for the curl command given we probably should warn the developer that it would remove all indices in ES cluster (so they should not do it if they use a shared ES cluster) Also we might want to provide more detailed help on how to remove specific indices for their MindMeld applications to free up ES cluster without having to remove everything.